### ISSUE TITLE
Installation is failing

### ISSUE THREAD

**Issue by sambhav-gore** [gooddata/gooddata-react-components#217]
Installation fails as it is unable to find `@gooddata/typings` module:

~~~
> npm install @gooddata/react-components

npm ERR! code E404
npm ERR! 404 Not Found: @gooddata/typings@0.0.4
~~~

the typings module is not even showing up in npmjs.

**Comment by pbenes** [comment]
Please check now, should be fixed (recently released `v4.0.1`, compatible with R153 GD platform update)

**Comment by sambhav-gore** [comment]
It works now. This issue may be closed.
